### ISSUE TITLE
Add "Roundess" slider to label customization

### ISSUE THREAD

**Issue by dumbeau** [dumbeau/AutoHotPie#51]
Some people like rounded rectangles and circles.

![image](https://user-images.githubusercontent.com/36665762/174325536-29923b39-5fe6-491f-bb19-537f7635796c.png)

**Comment by ruible** [comment]
这将使得按钮显得非常的酷。

**Comment by ruible** [comment]
> 我们也能得到完​​整的馅饼菜单切片吗？像GTA V换武器系统哈哈
> 
> ![图片](https://user-images.githubusercontent.com/63365609/174610435-6f433507-5af9-4671-9bee-6446467c21f3.png)

这也是不错的方案。
薄或许这样也可以设置的话，建议增加勾选项，要按钮更加的丰富。

**Comment by dumbeau** [comment]
> Can we also get complete pie menu slices? Like GTA V weapon changing system haha

Yeah that would be neat, but I'm not going to work on this one yet.  This would require a pretty big visuals rewrite and some addition of "Themes".  Although once that's added it would make it easier to add other types of visuals.  I'll keep this in mind but I'm going to stick to the roundness slider for now.

**Comment by ruible** [comment]
> > 我们也能得到完​​整的馅饼菜单切片吗？像GTA V换武器系统哈哈
> 
> 是的，那会很整洁，但我还不打算研究这个。这将需要相当大的视觉效果重写和一些“主题”的添加。虽然一旦添加了它会更容易添加其他类型的视觉效果。我会记住这一点，但我现在要坚持使用圆度滑块。

YES，支持薄的计划。
我们的按钮蒋变得非常完美。

**Comment by dumbeau** [comment]
Roundness slider has been added in v1.0.20, Feel free to open a new issue if anybody wants to talk about adding "Themes" to AHP.

**Comment by ruible** [comment]
这一次是成功的，对于只有图标按钮的时候，显得非常的酷。







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "dumbeau/AutoHotPie"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月30日(星期四) 下午5:47
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [dumbeau/AutoHotPie] Add "Roundess" slider to label customization (Issue #51)





 
Roundness slider has been added in v1.0.20, Feel free to open a new issue if anybody wants to talk about adding "Themes" to AHP.
